ui/components/src/context/PersesThemeProvider.tsx

```diff
@@ -0,0 +1,195 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { useTheme, ThemeOptions as MaterialThemeOptions } from '@mui/material';
+import merge from 'lodash/merge';
+import { EChartsTheme, PersesChartsTheme } from '../model';
+import { ChartsThemeProvider } from './ChartsThemeProvider';
+
+export interface PersesThemeProviderProps {
+  themeName: string;
+  muiTheme?: MaterialThemeOptions;
+  themeOverrides?: EChartsTheme;
+  children?: React.ReactNode;
+}
+
+export function PersesThemeProvider(props: PersesThemeProviderProps) {
+  const { children, themeName, themeOverrides } = props;
+  const muiTheme = useTheme();
+
+  const ltGrey = muiTheme.palette.grey[300] ?? '#dee2e6';
```

Comment:
Should we consider defining constants for these colors? One use case is if we want the same grey in other themes, we don't have to look up the specific grey values